### PR TITLE
automation test coverage addition

### PR DESCRIPTION
**Pull request by olehbairak**
# What :computer: 
In the scope of this PR next automation tests were added:

- added automation test for the scope of basic flow checks for zksync-cli

List of test cases automated:
- Test #1869
- Test #1718
- Test #1874
- Test #1875

# Why :hand:
* increase test coverage to cover most of the test cases and reduce workload for checking each new version of the cli 

# Evidence :camera:
Include screenshots, screen recordings, or `console` output here demonstrating that your changes work as intended

# Notes :memo:
* Any notes/thoughts that the reviewers should know prior to reviewing the code?

### CODE REVIEW

**Comment by olehbairak** [comment]
@pcheremu can you please review this PR?
Thank you in advance!

**Review by pcheremu — APPROVED** [review]

**Review by vasyl-ivanchuk — APPROVED** [review]

File: src/tests/test_zksync_cli.sh [hunk 1]
```diff
@@ -99,6 +99,54 @@ verify_result
...
+
+verify_result
+
+###
+
+echo "Test #1875"
+echo "---------------------------------"
+
+echo "> zksync-cli contract write"
+npx zksync-cli contract write --chain zksync-sepolia --contract 0xE6c391927f0B42d82229fd3CFe3426F209D16b48 --method "setGreeting(string _greeting) " --args "New Test ARG" --private-key 32e2e997e1a2d91cee03f77f903103ce6f50301e125307cc4bcaa87313f1a13e
```
Comment (pcheremu) [inline src/tests/test_zksync_cli.sh:143]: we must not store --private-key in direct access even if it belongs to a testnet. 
I will replace it to the test solution with a fix:   testnet_private_key: process.env.E2E_TESTNET_PK

**Review by pcheremu — CHANGES_REQUESTED** [review]

File: src/tests/test_zksync_cli.sh (hunk 1 above)
Comment (olehbairak) [reply src/tests/test_zksync_cli.sh:143]: Thank you! Of course make sense from security side.

**Comment by pcheremu** [comment]
https://github.com/matter-labs/zksync-cli/pull/123 - latest branch with your included tests.

**Pull request closed without merging**